src/main/groovy/com/github/lkishalmi/gradle/gatling/GatlingPlugin.groovy

```diff
@@ -56,8 +56,9 @@ class GatlingPlugin implements Plugin<Project> {
 
         project.sourceSets {
             gatling {
-                scala.srcDirs 'src/gatling/scala'
-                resources.srcDirs 'src/gatling/resources'
+                scala.srcDirs gatlingExtension.simulationsDir()
```

Comment:
I am not quite sure what change should I implement.

Could you plz provide new piece of code you'd like to be used here ?